### DISCUSSION TITLE
Log results

### DISCUSSION THREAD

**Discussion by kiatchun** [ThioJoe/YT-Spammer-Purge#197] [Q&A] [answered]
Is there a reason why the log files are in the .rtf format? I thought log files are in the .log format.

**Comment by ThioJoe** [comment, marked as answer]
> Is there a reason why the log files are in the .rtf format? I thought log files are in the .log format. 

Because it allows colors

**Reply by TechStudent10** [reply]
@kiatchun uh maybe mark this as answered...
